Rakefile

```diff
@@ -12,6 +12,12 @@ task run_processors: :environment do
   tracker = EventSourceryTodoApp.tracker
   db_connection = EventSourceryTodoApp.projections_database
 
+  # Need to disconnect before starting the processors
```

Comment:
It's so that the forked processes have their own connection / fork safety. https://github.com/jeremyevans/sequel/issues/691#issuecomment-21795190 You'll find the same for active record in `config/unicorn.rb` in Rails apps that use unicorn.